docker/k8s/Dockerfile

```diff
@@ -56,5 +56,5 @@ COPY --from=base $VTTOP/config/mycnf/backup.cnf /vt/config/mycnf/
 COPY --from=base $VTTOP/config/mycnf/rbr.cnf /vt/config/mycnf/
 
 # add vitess user and add permissions
-RUN groupadd -r --gid 999 vitess && useradd -r -g vitess --uid 999 vitess && \
+RUN groupadd -r --gid 2000 vitess && useradd -r -g vitess --uid 1000 vitess && \
```

Comment:
Whats the driver to change from 999 ? All the other containers have the same vitess user ( not that it matters one way or the other, just wanted to know if here was a driver for it )

Reply:
I wish I had remembered that originally when setting up the default pod security context. Anyone who has been using the helm charts has by default been running as 1000:2000 and I didn't want to silently break anyone upgrading.

```
securityContext:
  runAsUser: 1000
  fsGroup: 2000
```

https://github.com/vitessio/vitess/blob/master/helm/vitess/templates/_helpers.tpl#L64-L71